providers/summary/clearlydefined.js

```diff
@@ -132,20 +135,35 @@ class ClearlyDescribedSummarizer {
     let licenses = licenseUrls.map(extractLicenseFromLicenseUrl).filter(x => x)
     if (!licenses.length) licenses = licenseNames.map(x => SPDX.lookupByName(x) || x).filter(x => x)
     if (licenses.length) setIfValue(result, 'licensed.declared', SPDX.normalize(licenses.join(' OR ')))
+    const registryUrl = buildRegistryUrl(coordinates)
+    const versionUrl = buildVersionUrl(coordinates)
+    const downloadUrl = buildDownloadUrl(coordinates)
+    if (get(result, 'described'))
```

Comment:
The if is not needed. We want to set these values regardless of whether there is a `described` prop. Setting will create the `described` prop

Reply:
I've changed this after updating tests. 
Looks like that in some cases the described prop should be undefined at all, like `expect(summary.described).to.be.undefined`

Reply:
We were likely expecting undefined in the tests because previously there was nothing being added to `described` so there should not have been one. Now we have changed the system to add in URLs so the expectation should change.